.github/workflows/compiler_publish_docker_images.yml

```diff
@@ -116,7 +116,15 @@ jobs:
     runs-on: ${{ github.event.inputs.runner_name }}
     env:
       IMAGE: ghcr.io/zama-ai/cuda
-      TAG: "12-3"
+    strategy:
+      matrix:
+        include:
+          - name: cuda-12-3
+            tag: 12-3
+            dockerfile: docker/Dockerfile.cuda-123-env
+          - name: cuda-11-8
+            tag: 11-8
+            dockerfile: docker/Dockerfile.cuda-118-env
```

Comment:
It could be good to add 12.1 as well

Reply:
Preferably not for now as I'm rushing to have something working for the release, but we should definitely have all supported versions in here.